core/src/main/java/com/oath/oak/ValueUtils.java

```diff
@@ -1,368 +1,58 @@
 package com.oath.oak;
 
-import sun.misc.Unsafe;
-import sun.nio.ch.DirectBuffer;
-
 import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
-import java.util.AbstractMap;
-import java.util.Map;
 import java.util.function.Consumer;
 import java.util.function.Function;
 
-import static com.oath.oak.Chunk.VALUE_BLOCK_SHIFT;
-import static com.oath.oak.Chunk.VALUE_LENGTH_MASK;
-import static com.oath.oak.ValueUtils.LockStates.*;
-import static com.oath.oak.ValueUtils.ValueResult.*;
-import static java.lang.Integer.reverseBytes;
-
-// All of the methods are atomic (this atomicity is achieved using read/write locks)
-public class ValueUtils {
-
-    enum LockStates {
-        FREE(0), LOCKED(1), DELETED(2), MOVED(3);
-
-        public int value;
-
-        LockStates(int value) {
-            this.value = value;
-        }
-    }
+public interface ValueUtils {
 
     enum ValueResult {
-        SUCCESS, FAILURE, MOVED
-    }
-
-    private static final int LOCK_MASK = 0x3;
-    public static final int VALUE_HEADER_SIZE = 4;
-
-    private static Unsafe unsafe = UnsafeUtils.unsafe;
-
-    /**
```

Comment:
I do not understand why do you remove the comments when moving to interface... Each interface must be easily understandable. In my philosophy there is no such a thing as "not needed comment", even if a comment is obvious. Unless the comment is wrong, but then just correct the comment... Please add comments to the interface